libs/wire-api/src/Wire/API/Routes/Public/Brig.hs

```diff
@@ -101,7 +101,7 @@ instance AsUnion DeleteSelfResponses (Maybe Timeout) where
   toUnion Nothing = Z (I ())
   fromUnion (Z (I ())) = Nothing
   fromUnion (S (Z (I (DeletionCodeTimeout t)))) = Just t
-  fromUnion (S (S x)) = case x of {}
+  fromUnion (S (S x)) = case x of
```

Comment:
Is this because `x :: Void`?  can't you just skip this case, then?  (it's nice to have it there explicitly, just trying to understand.)

Reply:
Annoyingly, this change was made by my code formatter, but it does not match the formatting that is expected on CI. I reverted the change.